cytotable/convert.py

```diff
@@ -292,6 +292,73 @@ def _prepend_column_name(
     return table_path
 
 
+@python_app
+def _cast_data_types(
+    table_path: str, data_type_cast_map: Optional[Dict[str, str]] = None
+) -> str:
+    """
+    Cast data types per what is received in cast_map.
+
+    Example:
+    - table_path: parquet file
+    - data_type_cast_map: {"float": "float32"}
+
+    The above passed through this function will rewrite the parquet file
```

Comment:
When you say "rewrite", do you mean that cytotable will incur costs of writing the same file twice? Is there a way to set the initial write to happen using the `data_type_cast_map`?

Reply:
Thanks for this comment! Marking this as resolved as new updates no longer include a data rewrite.